src/test/java/dinkplugin/notifiers/LevelNotifierTest.java

```diff
@@ -97,7 +97,7 @@ void testNotifyTwo() {
             PRIMARY_WEBHOOK_URL,
             false,
             NotificationBody.builder()
-                .text(PLAYER_NAME + " has levelled Agility to 5 and Attack to 99")
+                .text(PLAYER_NAME + " has levelled Agility to 5 and Hunter to 99")
```

Comment:
I noticed that these tests don't actually verify the text on `NotificationBody` since it's a transient field. When running the tests with non-transient text, I found that this test was failing. I won't mess with `NotificationBody` since I guess there's a good reason why it uses transient

Reply:
Good catch, `transient` is important but we should add `@EqualsAndHashCode.Include` on the field